example/grpc_server.rb

```diff
@@ -0,0 +1,54 @@
+require 'graphql'
+require 'grpc'
+require 'json'
+require_relative './gen/federation_api_pb'
+require_relative './gen/federation_api_services_pb'
+
```

Comment:
Let's call this file graphql_grpc_server

Reply:
This is just a proof of concept 😉